ui/src/app/components/object-detail/object-detail.component.html

```diff
@@ -442,6 +442,42 @@ <h3 class="title">
       </div>
     </mat-tab>
 
+    <mat-tab
```

Comment:
The UI is showing convert to foreign key even though it's already foreign key : https://drive.google.com/corp/drive/folders/1tuiRpopwCcvAgqxYnSO6ZUr-j1g5WBJq

Reply:
And if its interleaved we should have some way of showing that it's interleaved and not under foreign keys

Reply:
Moved to backlog(no. 74) as foreign key and interleave UI design is yet to be finalised.